app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):

```

Comment:
![image](https://user-images.githubusercontent.com/103201010/212341444-fb15fbb2-19bb-4b66-be83-74f449527cb1.png)
fix this